Automattic/wp-calypso
# Framework: Upgrade to Webpack 4  (take two)

Reverts Automattic/wp-calypso#24006

**What broke?**
Support User support.

**Testing instructions**
- build docker locally and run it.
- use the support-user tool and the wpcalypso environment.
- check other server rendered pages and ensure they don't rely on function names in the same way
